db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
```

Comment:
- [ ] Kindly store your 5 random greetings, so that when a User starts your app the seeded database will be displayed, this is part of the requirement 👍🏿